includes/namespace.php

```diff
@@ -112,7 +113,7 @@ function remove_site( $error, $site ) {
 		return;
 	}
 	$site_id = (int) $site->blog_id;
-	$story   = new Story_Post_Type( new Experiments(), new Meta_Boxes() );
+	$story   = new Story_Post_Type( new Experiments(), new Meta_Boxes(), new Decoder() );
 	switch_to_blog( $site_id );
 	$story->remove_caps_from_roles();
```

Comment:
Similar:
```suggestion
	switch_to_blog( $site_id );
	Services::get( 'story_post_type' )->remove_caps_from_roles();
```

Reply:
Let's handle this here https://github.com/google/web-stories-wp/issues/5062